components/CalEPD/include/dke/depg750bn.h

```diff
@@ -0,0 +1,74 @@
+// DKE product https://dke.top/products/dke-e-ink-103-color-displays-large-screen-display-102-inch-resolution-is-960x640
```

Comment:
This part confuses me a little. Is the link the correct one for this display? Because it goes to a color epaper, different resolution

Reply:
Sorry this was a mistake. This is not the link to the added display. This comment has been removed.